sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -1193,7 +1193,7 @@ def test_sincos_rewrite_sqrt():
     for p in [1, 3, 5, 17]:
         for t in [1, 8]:
             n = t*p
-            for i in range(1, (n + 1)//2 + 1):
+            for i in range(1, min((n + 1)//2 + 1, 10)):
```

Comment:
I'm confused.  It seems like `p` and `t` are not used.  Why not rewrite the outer loop to be over `n in [1, 3, 5, 8]`?